modules/react/loading-dots/stories/examples/Accessible.tsx

```diff
@@ -1,6 +1,39 @@
 import React from 'react';
 import {LoadingDots} from '@workday/canvas-kit-react/loading-dots';
+import {canvas} from '@workday/canvas-kit-react/tokens';
```

Comment:
We need to use new tokens:  `import {system} from '@workday/canvas-tokens-web'` instead of old package.